tests/ui_tests/test_admin_page.py

```diff
@@ -822,3 +824,30 @@ def test_policy_permission_to_run_cluster_action_and_view_task(
             JobPageStdout(app_fs.driver, app_fs.adcm.url, 1).wait_page_is_opened()
 
     # pylint: enable=too-many-locals
+
+    @pytest.mark.usefixtures("login_to_adcm_over_api")
+    def test_policy_with_maintenance_mode(self, sdk_client_fs, app_fs, another_user, create_cluster_with_component):
+        """Test create a group on /admin/policies"""
+
+        _, _, host, _ = create_cluster_with_component
+        with allure.step("Create test role"):
+            test_role = sdk_client_fs.role_create(
+                name=CUSTOM_ROLE_NAME,
+                display_name=CUSTOM_ROLE_NAME,
+                child=[{"id": sdk_client_fs.role(name="Manage Maintenance mode").id}],
+            )
+        with allure.step("Create test policy"):
+            sdk_client_fs.policy_create(
+                name="Test policy",
+                role=test_role,
+                user=[sdk_client_fs.user(username=another_user['username'])],
+                objects=[host],
+            )
+        login_page = LoginPage(app_fs.driver, app_fs.adcm.url).open()
+        login_page.login_user(**another_user)
```

Comment:
Why do you want to explicitly login via UI?

Reply:
yep, changed to login by api

Reply:
Good, but why do you want to login not in fixture?
Is there a reason to do it after the policies are created?

Reply:
need to relogin by new user after new policy creation

Reply:
or reenter the page?
Do you mean that permissions are read only on login?
hmm...

Reply:
i need to change user and check that policy is working for this user

Reply:
got it, okay